### PR TITLE
Add support for UTF-8 character type

### LINKED ISSUE #79: Add support for UTF-8 character type
This is just a matter of mapping the SMPTE Type UL to [Java Charset name](https://docs.oracle.com/javase/7/docs/api/java/nio/charset/Charset.html).

### PR DESCRIPTION
**Pull request by mglanville**
This is just a matter of mapping the SMPTE Type UL to Java Charset name.

Resolves #79

### CODE REVIEW

File: regxmllib/src/main/java/com/sandflow/smpte/regxml/FragmentBuilder.java [hunk 1]
```diff
@@ -566,6 +567,8 @@ private void readCharacters(InputStream value, CharacterTypeDefinition definitio
             in = new InputStreamReader(value, "UTF-16BE");
         } else if (definition.getIdentification().equals(Char_UL)) {
             in = new InputStreamReader(value, "US-ASCII");
+        } else if (definition.getIdentification().equals(UTF8Character_UL)) {
+            in = new InputStreamReader(value, "UTF-8");
```
Comment (palemieux) [inline regxmllib/src/main/java/com/sandflow/smpte/regxml/FragmentBuilder.java]: Ok. I think we should add the following note:
```
/* NOTE: UTF-8 character encoding is not specified in ST 377-1*/
```
Also, have you checked with @thomasheritage ? I recall this topic of character encoding being discussed at length.

**Comment by mglanville** [comment]
It was @thomasheritage who asked me to make this change. I'll let him reply with his reasoning, but briefly:

While UTF-8 is not specified in ST 377-1, it is used in [RP 2057](http://ieeexplore.ieee.org/servlet/opac?punumber=7290394). We'd like to be able to generate a RegXML document for an MXF file that uses RP 2057 to carry UTF-8 text.

**Comment by palemieux** [comment]
> While UTF-8 is not specified in ST 377-1, it is used in RP 2057. We'd like to be able to generate a RegXML document
> for an MXF file that uses RP 2057 to carry UTF-8 text.

Right. So,

* is there a small file that can be added as a unit test?
* the comment should
```
/* NOTE: UTF-8 character encoding is specified in RP 2057 */
```

**Comment by mglanville** [comment]
I'll generate a sample file and add it as a test.

**Comment by thomasheritage** [comment]
Thanks, this all looks very good.
Although ST 377-1 doesn't mention UTF-8 it does explain that Strings are concatenations of any sort of characters (giving UTF-16 as one example). UTF-8 is also used elsewhere (such as RDD 18).

**Comment by thomasheritage** [comment]
Apologies for delaying this. I think this is ready to be merged in now.

**Comment by palemieux** [comment]
Replaced by #82

**Pull request closed without merging**